spec/serializers/v1/billable_metric_serializer_spec.rb

```diff
@@ -49,4 +50,11 @@
 
     expect(result['billable_metric']['draft_invoices_count']).to eq(1)
   end
+
+  it 'returns the number of plans' do
+    plan = create(:plan, organization: billable_metric.organization)
+    charge = create(:standard_charge, billable_metric:, plan:)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `charge`. (https://rubystyle.guide#underscore-unused-vars)